Policies/github-terms-of-service.md

```diff
@@ -257,7 +257,7 @@ You are responsible for all fees, including taxes, associated with your use of t
 It is your responsibility to properly cancel your account with GitHub. You can [cancel your account at any time](/articles/how-do-i-cancel-my-account/) by going into your Settings in the global navigation bar at the top of the screen. The Account screen provides a simple, no questions asked cancellation link. We are not able to cancel accounts in response to an email or phone request.
 
 #### 2. Upon Cancellation
-We will retain and use your information as necessary to comply with our legal obligations, resolve disputes, and enforce our agreements, but barring legal requirements, we will delete your full profile and the Content of your repositories within 90 days of cancellation or termination (though some information may remain in encrypted backups). This information can not be recovered once your account is cancelled.
+We will retain and use your information as necessary to comply with our legal obligations, resolve disputes, and enforce our agreements, but barring legal requirements, we will delete your full profile and the Content of your repositories within 90 days of cancellation or termination (though some information may remain in encrypted backups). This information can not be recovered once your account is canceled.
```

Comment:
Both spellings are correct in English. [Cancelled](https://en.wiktionary.org/wiki/cancelled) is the primary spelling, with [canceled](https://en.wiktionary.org/wiki/canceled) as an alternative.